### ISSUE TITLE
OAuth client configuration is not persisted until mint bucket is defined

### ISSUE THREAD

**Issue by lsavolainen** [zalando-stups/yourturn#506]
In an app, if you try to configure the OAuth client section before a mint bucket has been set in access control section the changes are lost.

**Comment by prayerslayer** [comment]
Yes, still the case, see https://github.com/zalando-stups/yourturn/issues/414
